### ISSUE TITLE
Extension has recently started causing playlist pages to go into an "infinite loading loop"

### ISSUE THREAD

**Issue by grandslammer** [sridhama/youtube-playlist-duration#6]
- Issue has only started recently (last few days!)
- To replicate: Load any playlist page on YouTube. The page keeps trying to load. Might be connected to disable polymer in the URL?
- Disabling the extension causes the issue to stop, but obviously the extension cannot be used when its disabled

**Comment by rohitkrishna094** [comment]
Yes I am having this problem as well. I think this extension redirects the baseurl and adds a query param `disable_polymer=true`. This worked till now. 

However try opening this in incognito(i.e., without the extension), and it redirects back to normal page. I am guessing Youtube made an update to redirect back to main url whenever this query param is added. However this extension also adds this back again and therefore we have an infinite loop.

I think we might need to redo the logic and not add this query param since Youtube is no longer honoring it.
